functions.php

```diff
@@ -56,4 +58,25 @@ function displayPokemon(array $pokemons): string {
         }
     }
     return $result;
-}
\ No newline at end of file
+}
+
+/**
+ * Adds data to the database
+ *
+ * @param none
+ *
+ * @return array, when returned the data will be sent to the database
+ */
+function addPokemonToDatabase()
+{
+    $dataBase = getDatabaseObject();
+    $pokedexNumber = $_POST['addPokedexNo'];
+    $pokemonName = $_POST['addPokemonName'];
+    $pokemonType = implode(' ', $_POST['addPokemonType']);
+    $pokemonImage = $_POST['addPokemonImage'];
+    $insert = $dataBase->prepare('INSERT INTO `kanto_pokedex` (`pokedex_no`, `name`, `type`, `img_source`) VALUES (?, ?, ?, ?);');
+    $insert->execute([$pokedexNumber, $pokemonName, $pokemonType, $pokemonImage]);
```

Comment:
no check to see if your query worked?

Reply:
as in unit test or just a message to let the user know?

Reply:
a message to let the user know

Reply:
updated